.ci/test.sh

```diff
@@ -113,12 +114,14 @@ if [[ $TASK == "lint" ]]; then
         'r-lintr>=3.1.2'
     # shellcheck disable=SC1091
     source activate "${CONDA_ENV}"
-    echo "Linting Python code"
-    bash ./.ci/lint-python.sh || exit 1
+    echo "Linting Python and bash code"
+    bash ./.ci/lint-python-bash.sh || exit 1
```

Comment:
Due to [recently added `shellcheck`](https://github.com/microsoft/LightGBM/pull/6685) in pre-commit hook.